### PR TITLE
#166237752- Configure ESLint with AirBnB style Guide

### PR DESCRIPTION
**Pull request by chialuka**
#### What does this PR do?

- Configure ESLint with AirBnB style guide

- Add .babelrc file for babel preset

- Add PR template to automate PR descriptions

- Fix linting errors

#### Description of Task to be completed?

- Ensure project is configured to run on the ESLint linter 

#### How should this be manually tested?

- Clone the repo and from the root run `npm run start-dev` to start the development server

#### Any background context you want to provide?
Every distributed team needs a style guide that will help team members work in sync. This PR provides the style guide this project will use

#### What are the relevant pivotal tracker stories?
[#166237752](https://www.pivotaltracker.com/story/show/166237752)

### CODE REVIEW

**Review by tolulope-od — APPROVED** [review]
Good stuff, @LukasChiama. These your ESLint rules seem very strict. LOL! but we'll stick to them. Great job once again

**Comment by ayodejiAA** [comment]
Awesome work. @LukasChiama

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-env"
+  ]
+}
```
Comment (abdulfatai360) [inline .babelrc]: @LukasChiama  Put a new line after this LOC... I expected ESLint (AirBnB) to flag it if there are no new lines at the end of any file

File: .hound.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc.json
```
Comment (abdulfatai360) [inline .hound.yml]: @LukasChiama  Add new line here

Comment (chialuka) [reply .hound.yml]: That's a .yml file. White spaces matter in .yml files. Adding a new line, which is white space, could be counter productive

File: .babelrc (hunk 1 above)
Comment (chialuka) [reply .babelrc]: Okay. ESLint does not check configuration files for errors and this is a configuration file.

**Review by abdulfatai360 — APPROVED** [review]
Awesome @LukasChiama . Approved!

**Review by ayodejiAA — APPROVED** [review]
Good work.

File: .hound.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
Comment (akpante3) [inline .hound.yml:3]: add new-line

**Review by akpante3 — APPROVED** [review]

**Pull request merged**